src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -99,6 +100,12 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
             {
                 this.meter.CreateObservableCounter($"{metricPrefix}assembly.count", () => (long)AppDomain.CurrentDomain.GetAssemblies().Length, description: "Number of Assemblies Loaded.");
             }
+
+            if (options.IsExceptionsEnabled)
+            {
+                AppDomain.CurrentDomain.FirstChanceException += (source, e) => ++this.exceptionCount;
```

Comment:
Additional qn to all : Do we think this metric is useful without any dimensions? I think at least the name of the Exception should be an added as a tag.

And if we do that, we need to make this a Counter, not ObservableCounter.

Reply:
@reyang Thoughts on this?

Reply:
I guess the main scenario is to get a rough idea of the total number and see if it is abnormally high. If the intention is to breakdown based on exception type, I would expect something more context - e.g. being able to group exceptions based on inheritance hierarchy, which seems to go beyond the current scope.

Reply:
sounds like a good plan to start with just counter.

Reply:
I've changed it to Counter.